### PR TITLE
Update to fd.o 22.08

### PR DESCRIPTION
**Pull request by orowith2os**
https://github.com/flathub/com.scoutshonour.dtipbijays/issues/6

### CODE REVIEW

**Comment by orowith2os** [comment]
bot, build

**Comment by bbhtt** [comment]
What was the failure here? If it was pygame, replace pygame-1.9.6 with shared-modules. That one should hopefully work.

Also, remove the unavailable arches from here https://github.com/flathub/com.scoutshonour.dtipbijays/blob/dc0d0edafa72eaa31d735e136a24e288c6172d5c/com.scoutshonour.dtipbijays.yaml#L48

Thanks

File: com.scoutshonour.dtipbijays.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 app-id: com.scoutshonour.dtipbijays
 runtime: org.freedesktop.Platform
-runtime-version: '20.08'
+runtime-version: '22.08'
```
Comment (bbhtt) [inline com.scoutshonour.dtipbijays.yaml:3]: Can you try 23.08?

Comment (orowith2os) [reply com.scoutshonour.dtipbijays.yaml:3]: Would you mind taking over? I'm not too interested in stretching myself out more, again.

Comment (bbhtt) [reply com.scoutshonour.dtipbijays.yaml:3]: Ok sure

**Comment by MateusRodCosta** [comment]
Closing in favor of #9.

**Pull request closed without merging**